mlrun/common/schemas/partition.py

```diff
@@ -46,24 +46,24 @@ def as_duration(self) -> timedelta:
             return timedelta(weeks=1)
 
     @classmethod
-    def from_function(cls, partition_function: str):
+    def from_function(cls, partition_expression: str):
```

Comment:
why not `expression` all the way then?
```suggestion
    def from_expression(cls, partition_expression: str):
```

Reply:
@TomerShor good point, will rename it